pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -261,13 +262,17 @@ def _available_devices(
     def _dict_to_backendinfo(cls, dct: Dict[str, Any]) -> BackendInfo:
         name: str = dct.pop("name")
         n_qubits: int = dct.pop("n_qubits")
+        n_cl_reg: Optional[int] = None
+        if "n_classical_registers" in dct:
```

Comment:
Should we expect this always to be there, and error if not, as for `name` and `n_qubits`, or is it genuinely optional?

Reply:
It is optional for the family devices( for example ("H1"), so I will keep it as it is?

Reply:
(You can take a look on the two commits if you want to see what I tried) @cqc-melf
[make n_cl_reg required](https://github.com/CQCL/pytket-quantinuum/pull/128/commits/2fb54f5b6ff41ea71096c2c439aae6dd78c75be3)
[2fb54f5](https://github.com/CQCL/pytket-quantinuum/pull/128/commits/2fb54f5b6ff41ea71096c2c439aae6dd78c75be3) and @cqc-melf
[Revert "make n_cl_reg required"](https://github.com/CQCL/pytket-quantinuum/pull/128/commits/68a259ceb5fce0fdb8699818c501a9a003117497)
[68a259c](https://github.com/CQCL/pytket-quantinuum/pull/128/commits/68a259ceb5fce0fdb8699818c501a9a003117497)

Reply:
OK, thanks for checking.